src/append_only_data.rs

```diff
@@ -109,20 +135,25 @@ impl UnpubPermissionSet {
     }
 }
 
+/// Set of published permissions for a user.
 #[derive(Copy, Clone, Serialize, Deserialize, PartialEq, PartialOrd, Ord, Eq, Hash, Debug)]
 pub struct PubPermissionSet {
+    /// `true` if the user can append.
```

Comment:
Should be
```suggestion
    /// `Some(true)` if the user can append.
```
And probably worth expanding on why it's an `Option` (i.e. the fact that we use permissions for `Anyone` if it's `None`)?